File: c++/Driver.cpp
#include <iostream>
#include "Driver.h"

using namespace std;

Driver::Driver(int driverId, string driverName, string phoneNumber,string licenseNumber, int experience,string availabilityStatus)
{
    this->driverId = driverId;
    this->driverName = driverName;
    this->phoneNumber = phoneNumber;
    this->licenseNumber = licenseNumber;
    this->experience = experience;
    this->availabilityStatus = availabilityStatus;
}

void Driver::displayDriverDetails()
{
    cout << "\n----------- Driver Details -----------" << endl;
    cout << "Driver ID : " << driverId << endl;
    cout << "Driver Name : " << driverName << endl;
    cout << "Phone Number : " << phoneNumber << endl;
    cout << "License Number : " << licenseNumber << endl;
    cout << "Experience : " << experience << " Years" << endl;
    cout << "Availability : " << availabilityStatus << endl;
}

File: python/Driver.py
class Driver:

    def __init__(
        self,
        driverId,
        driverName,
        phoneNumber,
        licenseNumber,
        experience,
        availabilityStatus
    ):
        self.__driverId = driverId
        self.__driverName = driverName
        self.__phoneNumber = phoneNumber
        self.__licenseNumber = licenseNumber
        self.__experience = experience
        self.__availabilityStatus = availabilityStatus

    def displayDriverDetails(self):

        print("\n----------- Driver Details -----------")
        print("Driver ID :", self.__driverId)
        print("Driver Name :", self.__driverName)
        print("Phone Number :", self.__phoneNumber)
        print("License Number :", self.__licenseNumber)
        print("Experience :", self.__experience, "Years")
        print("Availability :", self.__availabilityStatus)